app/models/miq_policy_content.rb

```diff
@@ -3,6 +3,53 @@ class MiqPolicyContent < ApplicationRecord
   belongs_to :miq_event_definition
   belongs_to :miq_action
 
+  def self.seed
+    fixture_file = File.join(FIXTURE_DIR, "miq_policy_contents.yml")
+    policy_contents = File.exist?(fixture_file) ? YAML.load_file(fixture_file) : []
+
+    policy_contents.each do |content|
+      find_associations!(content)
```

Comment:
I'd prefer this not modify the receiver so it's easier to test and not have side effects.  Maybe merge the result of this method.